Framework/Algorithms/src/FilterEvents.cpp

```diff
@@ -23,6 +23,7 @@
 #include "MantidKernel/System.h"
 #include "MantidKernel/TimeSeriesProperty.h"
 #include "MantidKernel/VisibleWhenProperty.h"
+#include "MantidGeometry/Instrument/Goniometer.h"
```

Comment:
The clang-format-5.0 fails for me on this line. It puts it further up to respect alphabetical ordering. How did you get it to build successfully?

Reply:
It seems that the build server was running an older version of clang-format. I think it's fixed now on the master https://github.com/mantidproject/mantid/commit/ba5d2a0daa843906e4f8b791865a3ccc5af97943